recipes/install.rb

```diff
@@ -1,16 +1,29 @@
 # Cookbook Name:: chef-grafana
 # Recipe:: default
 
-# The http apt method isn't always installed by default, and we have a https
-# repository. Make sure it's installed.
-package 'apt-transport-https'
+case node['platform_family']
+when 'debian'
+  # The http apt method isn't always installed by default, and we have a https
+  # repository. Make sure it's installed.
+  package 'apt-transport-https'
 
-apt_repository 'grafana' do
-  uri 'https://packagecloud.io/grafana/stable/debian/'
-  distribution 'wheezy' # This is used for all debian/ubuntu versions
-  key 'https://packagecloud.io/gpg.key'
-  components ['main']
-  action :add
+  apt_repository 'grafana' do
+    uri 'https://packagecloud.io/grafana/stable/debian/'
+    distribution 'wheezy' # This is used for all debian/ubuntu versions
+    key 'https://packagecloud.io/gpg.key'
+    components ['main']
+    action :add
+  end
+when 'rhel'
+  yum_repository 'grafana' do
+    description 'Grafana - Stable'
+    baseurl 'https://packagecloud.io/grafana/stable/el/$releasever/$basearch'
```

Comment:
$releasever requires centos-release package. In the yum cookbooks we've been updating to use #{node['platform_version'].to_i} instead.